### PR TITLE
6.GET

### PR DESCRIPTION
**Pull request by NadiaIb**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -115,3 +115,40 @@ describe.only("/api/reviews", () => {
...
+  });
+test("GET -status:404 - valid ID but non-existent review ID", () => {
+  return request(app)
+    .get("/api/reviews/123/comments")
+    .expect(404)
+    .then((response) => {
+      expect(response.body.msg).toBe("not found");
+    });
+});
+});
```
Comment (robert112233) [inline __tests__/app.test.js:154]: Missing 200 test, valid article that just has no comments (returns empty array []) 

You may need a checkArticleExists function to differentiate between valid articles with no comments and invalid articles with no comments

**Review by robert112233 — APPROVED** [review]
Great work Nadia, Add the additional test & checkExists and submit another PR for review 👍
